packages/consumption/src/modules/notifications/NotificationsController.ts

```diff
@@ -33,8 +33,8 @@ export class NotificationsController extends ConsumptionBaseController {
         return LocalNotification.from(notification);
     }
 
-    public async sent(message: Message): Promise<LocalNotification> {
-        if (!message.isOwn) throw new Error("Cannot send a Notification from a foreign message.");
+    public async saveSent(message: Message): Promise<LocalNotification> {
```

Comment:
Why was the name of the `sent` method of the `NotificationsController` adjusted? Why wasn't the name of the `received` method adjusted as well?

Reply:
see above.

Reply:
I don't see anything above. I don't like the new naming.

Reply:
Additionally: this is nothing which should pop up in a PR that is dedicated to changing error codes and messages.

Reply:
In the other deleted PR @Milena-Czierlinski wrote this:

_The error message seems misleading to me. If I understand it correctly, the use case SentNotification is meant to save a LocalNotification for a Notification that was sent inside a Message by oneself. If the owner of the Message is not the Identity itself, this error is thrown. In this case, I'd expect an error message like The Message ${messageId} was received from a peer, but an own Message is expected here to save its Notification content..

Besides, also the naming of the SentNotificationUseCase seems not ideal to me, as it doesn't appropriately describe what happens inside of it. However, this is probably outside the scope of this PR._

 and after that we agreed about this way and so I changed it.

Reply:
`sent` is within our naming schema (we do the same for requests for example). I stay with my opinion that **if** there would be need for changing this 

1. this is the wrong PR to do so
2. we have to discuss this first

therefore - please undo that everywhere.

Reply:
Ok, than I will undo this.